ext/readline/readline.c

```diff
@@ -181,7 +181,7 @@ PHP_FUNCTION(readline_info)
 		add_assoc_long(return_value,"attempted_completion_over",rl_attempted_completion_over);
 	} else {
 		if (zend_string_equals_literal_ci(what,"line_buffer")) {
-			oldstr = rl_line_buffer;
+			oldstr = strdup(rl_line_buffer ? rl_line_buffer : "");
```

Comment:
The `strdup("")` doesn't seem to be necessary (that's already handled by `SAFE_STRING()`. It's not wrong though, and performance is likely irrelevant here.